README.md

```diff
@@ -323,6 +325,23 @@ run {
 }
 ```
 
+Using the ModularJavaExec task
+===
+The `application` plugin can handle only one executable application.
+To start multiple applications, you typically need to create a `JavaExec` task for each executable application. 
+The module plugin offers a similar task named `ModularJavaExec`, which helps executing modular applications.
+This task automatically configures the JVM with the correct arguments such as `--module-path`.
+It exposes the same properties and methods as the `JavaExec` task, the only difference being that
+the module name should also be provided when setting the `main` property.  
+
+```gradle
+task runDemo1(type: ModularJavaExec) {
+    group = "Demo"
+    description = "Run the Demo1 program"
+    main = "greeter.demo/demo.Demo1"
```

Comment:
Question: How would you provide extra jvm arguments? Should there be an example of this here?

Also, besides running, does it also create distributions? Or is that something only to be expected from the `application` plugin?

Reply:
Let's do that separately, because it seems like a non trivial addition. That way we can get this feature out first.